main.tf

```diff
@@ -183,6 +183,186 @@ EOF
   }
 }
 
+resource "aws_instance" "ext_tableau_linux_test" {
+  count                       = "${var.environment == "prod" ? "1" : "0"}"                       # Allow different instance count in prod and notprod
+  key_name                    = "${var.key_name}"
+  ami                         = "${data.aws_ami.ext_tableau_linux.id}"
+  instance_type               = "c5.2xlarge"
+  iam_instance_profile        = "${aws_iam_instance_profile.ext_tableau.id}"
+  vpc_security_group_ids      = ["${aws_security_group.sgrp.id}"]
+  associate_public_ip_address = false
+  subnet_id                   = "${aws_subnet.subnet.id}"
+  private_ip                  = "${element(var.dq_external_dashboard_instance_ip, count.index)}"
```

Comment:
This will pick up the same IP as the current live instance, so will fail.
On this occasion, set the IP as 10.1.14.116

Reply:
added IP